docs/reference/data-streams/lifecycle/tutorial-manage-data-stream-retention.asciidoc

```diff
@@ -0,0 +1,184 @@
+[role="xpack"]
+[[tutorial-manage-data-stream-retention]]
+=== Tutorial: Data stream retention
+
+preview::[]
+
+In this tutorial, we are going to go over the data stream lifecycle retention, define it, go over how it can be configured and how
+it can be applied. Keep in mind, the following options apply only to data streams that have are managed by the data stream lifecycle.
+
+
+////
+[source,console]
+----
+PUT /_index_template/template
+{
+  "index_patterns": ["my-data-stream*"],
+  "template": {
+    "lifecycle": {}
+  },
+  "data_stream": { }
+}
+
+PUT /_data_stream/my-data-stream
+----
+// TESTSETUP
+////
+
+////
+[source,console]
+----
+DELETE /_data_stream/my-data-stream*
+DELETE /_index_template/template
+DELETE /_data_stream/_global_retention
+----
+// TEARDOWN
+////
+
+You can verify this using the <<data-streams-get-lifecycle,get data stream lifecycle API>>:
+
+[source,console]
+--------------------------------------------------
+GET _data_stream/my-data-stream/_lifecycle
+--------------------------------------------------
+
+The result should look like this:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "data_streams": [
+    {
+      "name": "my-data-stream",                                   <1>
+      "lifecycle": {
+        "enabled": true                                           <2>
+      }
+    }
+  ]
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+<1> The name of your data stream.
+<2> Ensure that the lifecycle is enabled, meaning this should be `true`.
+
+. <<what-is-retention>>
```

Comment:
Nit: In the generated docs, these section header links look a bit out of place after the example showing how to verify that lifecycle is enabled. 

Maybe checking for lifecycle=true  could be its own section, or could just be place after the section header links and before the sections themselves?

Reply:
Nice, looks good like this. Definitely makes sense to leave out of header list